### DISCUSSION TITLE
GitHub.io website showing incorrect files (and thus webpage) on Chrome, but correct on other browsers

### DISCUSSION THREAD

**Discussion by jvherck** [community/community#36833] [Other Feature Feedback, Questions, & Ideas]
I made a small personal website. It works great and as intended when I run it on my pc! But then i upload it to GitHub and update GitHub Pages, and all of a sudden it's nothing like how it should be and showing source files of one of the earlier versions! I've been trying to solve this for 3 hours now! I went to look in the GitHub repo and it's showing the exact files as on my pc, which should in fact work perfectly! But that means that GitHub Pages is running an older version of the site (which has NEVER been uploaded to GitHub, I even deleted the repo AND the folder on my pc multiple times to see if that does anything but still the same). Then I opened Microsoft Edge to see if it's the same there, but when I go to the website there, it shows it correctly! Now someone please tell me how or why this is happening? Why is Chrome showing files that are not even ran by GitHub Pages? Because Microsoft Edge does show it correctly...

I'm so lost and been trying to fix this issue for hours and hours, someone please help me out.

**Comment by yoannchaudet** [comment]
Hello, if you have a link to share it would be much easier to understand/help :)

**Reply by jvherck** [reply]
Thank you for your very fast response, I expected it would take about a day to get a reply 😅
Anyway a few minutes after posting this the problem seemed to have resolved itself somehow... I'm guessing Chrome uses some kind of cache that stores certain files. So once it recognised my site it just used the files from earlier? I was using Chrome to develop and test the site, so what I'm saying is I guess it still had _those files_ cached somewhere and used those to show the GitHub Pages website instead of the actual GitHub Pages files...? Anyhow it does work now.
Thanks again for your fast reply!

**Comment by jfullstackdev** [comment]
it definitely is, so you might want to be sure with that next time by logging in Incognito mode

**Comment by ossanna16** [comment]
Thank you for asking a good question @jvherck, I'm glad you found our community 🙂
